src/SonarScanner.MSBuild.Tasks/WriteZeroLengthFiles.cs

```diff
@@ -26,7 +26,7 @@ namespace SonarScanner.MSBuild.Tasks
 {
     /// <summary>
     /// Build task to write out one or more zero-length files,
-    /// overwriting any existing files
+    /// overwriting any existing files.
```

Comment:
This PR shouldn't be changing content of the `src` directory

Reply:
All the changes in the src files were done to change code and make the ccov run to test that it is running as expected.

Reply:
On comments?